### DISCUSSION TITLE
Oryx detects wrong Python version

### DISCUSSION THREAD

**Discussion by davetapley** [community/community#73116] [Codespaces]
### Select Topic Area

Bug

### Body

My repo has `.python-version` with `3.11..4`, but when creating a new Codespace the log says:

```
Configuring codespace...
Running oryx...
$ python -m site --user-site
/home/codespace/.local/lib/python3.10/site-packages
python process exited with exit code 0
$ python --version
Python 3.10.8
python process exited with exit code 0
$ oryx build --manifest-dir "/workspaces/.oryx" --property packagedir="/home/codespace/.local/lib/python3.10/site-packages" --property python_version="3.10.8" --log-file "/workspaces/.oryx/build.log" "/workspaces/ng"
Operation performed by Microsoft Oryx, https://github.com/Microsoft/Oryx
You can report issues at https://github.com/Microsoft/Oryx/issues

Oryx Version: 0.2.0.0, Commit: , ReleaseTagName: 

Build Operation ID: 07fb2adb2065fde0
Repository Commit : dbfd8c667920bdfdf14d9d63d2f2c6d3f66e88b6
OS Type           : focal-scm
Image Type        : vso-focal

Detecting platforms...
Detected following platforms:
  python: 3.10.8
```

I guess it's using `3.10.8` because that is what's in the base image, but it is wrong, and very confusing.

**Comment by arienshibani** [comment]
A couple of things you can try out:

1. **Check the `.python-version`** File: Ensure that the .python-version file in your repository contains the correct version specification. It seems like it might not be recognized during the Codespace creation? You may want to explicitly specify the version without a range.

2. **Check the Codespace configuration**: Check the settings to see if there's any override or specific configuration that might be affecting the Python version. Look for any settings related to the default Python version in your Codespace configuration.

3. **Update Oryx:** Verify if there is a newer version of Oryx available. If you are using an outdated version, updating to the latest release might resolve compatibility issues.

Cached data might sometimes cause issues. Try clearing any cached data related to your Codespace or Oryx if all of the above fails to help you out.
